src/universal/modules/outcomeCard/containers/OutcomeCard/OutcomeCardContainer.js

```diff
@@ -36,6 +36,8 @@ class OutcomeCardContainer extends Component {
     super(props);
     const {outcome: {content}} = props;
     this.state = {
+      isEditingText: false,
+      isEditingMeta: false,
```

Comment:
ah, really well thought out!
you already started talking about my next question, which is how to keep the clients in sync.
In the screenshot below, you see 2 clients. the bottom right client is editing, but since he is editing meta, the top client doesn't see him as editing. can you fix the call to `this.annouceEditing` so the 2 clients are always in sync? 
![screenshot from 2017-08-16 18-09-59](https://user-images.githubusercontent.com/5514175/29391786-7f1570a4-82ae-11e7-9308-0f817290f99d.png)